src/room.ts

```diff
@@ -171,23 +173,89 @@ export class Room extends EventEmitter implements Sayable {
   }
 
   /**
-   * @todo document me
+   * Get room leave event, emitted when bot remove someone from the room or someone remove the bot from the room.
+   * If both personA and personB are not the bot itselt, the event cannot be fired if personA remove personB from the room
+   * This is Sayable for all listeners.
+   * Which means there will be a this.say() the method inside listener call,
+   * you can use it sending a message to filehelper, just for logging/reporting usage for your convenience.
+   * @param {'leave'} event
+   * @param {(this: Room, leaver: Contact) => void} listener
+   * @returns {this}
+   * @example
+   * ```ts
+   *  const room = await Room.find({topic: 'event-room'}) // change `event-room` to any room topic in your wechat
+   *  if (room) {
+   *    room.on('leave', (room: Room, leaverList: Contact[]) => {
+   *      const nameList = leaverList.map(c => c.name()).join(',')
+   *      console.log(`Room ${room.topic()} lost member ${nameList}`)
+   *    })
+   *  }
+   * ```
    */
   public on(event: 'leave', listener: (this: Room, leaver: Contact) => void): this
+
   /**
-   * @todo document me
+   * Get room join event, emitted when someone join the room
+   * This is Sayable for all listeners.
```

Comment:
So do you mean that if something is important, then we should put it everywhere?

I have to disagree with you.